monitoring/alerts/metricsAlerts/dev/asb-dev-westus2-AppEndpointDown.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
This file should be renamed this file to use westus3. The other central, and westus2 metricAlerts files should be renamed as well.